test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -207,6 +207,10 @@ class BalancerServiceImpl : public BalancerService {
   Status BalanceLoad(ServerContext* context, Stream* stream) override {
     // TODO(juanlishen): Clean up the scoping.
     gpr_log(GPR_INFO, "LB[%p]: BalanceLoad", this);
+    {
```

Comment:
We should add this check so that the immediate restart of an LB call doesn't increase the request count.